python/pfs/pipe2d/weekly/test_weekly.py

```diff
@@ -81,7 +81,7 @@ def testObjects(self):
 )
 class ArcTestCase(lsst.utils.tests.TestCase):
     def setUp(self):
-        self.butler = Butler(os.path.join(weeklyRerun, "calib", self.arms, "arc"))
+        self.butler = Butler(os.path.join(weeklyRerun, "calib", self.arms, "detectorMap"))
```

Comment:
Does this work? I don't see anything in `process_weekly.sh` with a rerun ending in `detectorMap`.

Reply:
When I wrote `generateCommands,py` , I chose not `arc` but `detectorMap` for the key name of the YAML block describing how to run `reduceArc.py` . So the name of this subdirectory is indeed `detectorMap` .